splinterd/Cargo.toml

```diff
@@ -81,6 +81,7 @@ experimental = [
     # The following features are experimental:
     "auth",
     "authorization",
+    "authorization-handler-rbac",
```

Comment:
Can we shorten this to `auth-handler-rbac` for brevity? The maintenance and allow_keys ones would especially benefit from this (they'll match). Can you also guard the handler itself in libsplinter?

Reply:
I think the name is correct, but I have added the guard to libsplinter as well.